### PR TITLE
Move item popup tabs to header so they don't scroll out on mobile

### LINKED ISSUE #9879: Item popup tabs scroll away
I think they should probably be fixed / part of the header, while their content scrolls?

### PR DESCRIPTION
**Pull request by robojumper**
Fixes #9879.

### CODE REVIEW

File: src/app/item-popup/ItemDetails.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ import ItemDescription from './ItemDescription';
 import styles from './ItemDetails.m.scss';
 import ItemExpiration from './ItemExpiration';
 import ItemPerks from './ItemPerks';
+import './ItemPopupBody.scss';
```
Comment (robojumper) [inline src/app/item-popup/ItemDetails.tsx:32]: This should be modularized at some point...

File: src/app/item-popup/ItemPopupBody.tsx [hunk 2]
```diff
@@ -1,91 +0,0 @@
...
-      component: <ItemTriage item={item} />,
-    });
-  }
-
-  return (
-    <div>
-      {/* TODO: Should these be in the details? Or in the header? */}
-      {item.percentComplete !== 0 && !item.complete && (
-        <div className="item-xp-bar" style={{ width: percent(item.percentComplete) }} />
-      )}
```
Comment (robojumper) [inline src/app/item-popup/ItemPopupBody.tsx:58]: `item-xp-bar` hasn't existed for over four years now as far as I can tell (46a59b2a2dcd27e87f8ca93f2860021373dc2bc8), resulting in an unstyled empty div, which I simply removed. This looks like it was providing a global completion progress bar on the item popup itself, in addition to the bars for every objective...

Comment (bhollis) [reply src/app/item-popup/ItemPopupBody.tsx:58]: I wonder if that was a D1 thing...

**Review by bhollis — APPROVED** [review]
Code looks good!

**Pull request merged**